cpp/tests/large_strings/large_strings_tests.cpp

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Humnn, it seems that this pair of header/source are actually not doing any test but they have the suffix `_tests`. Maybe `large_string_utils.*` or `large_string_test_utils.*`?